backend/Functions/Edna.Connect/Lti1Api.cs

```diff
@@ -78,10 +78,23 @@ private Assignment ConvertRequestToAssignment(LtiRequest ltiRequest)
                 ResourceLinkId = ltiRequest.ResourceLinkId,
                 Name = ltiRequest.ResourceLinkTitle,
                 CourseName = ltiRequest.ContextTitle,
-                LtiVersion = ltiRequest.Version,
+                LtiVersion = GetLtiVersion(ltiRequest),
                 ContextMembershipsUrl = ltiRequest.Parameters.FirstOrDefault(pair => pair.Key == "custom_context_memberships_url").Value,
                 OAuthConsumerKey = ltiRequest.ConsumerKey
             };
         }
+
+        private static string GetLtiVersion(LtiRequest ltiRequest)

```

Comment:
Wondering if we should be moving this to `LtiVersion` Enum?

Reply:
LtiRequest for LTI1 and LTI1.3 is different, so moving this function to the enum won't be useful

Reply:
Apologies, I don't understand. Are we saying that string values are different for mapping to the same enum value? or is it something else?

Reply:
the request type is different for lti 1 and lti 1.3
For LTI1, the request is of the type `LtiLibrary.NetCore.Lti.v1.LtiRequest` whereas for LTI1.3, the request type is `System.IdentityModel.Tokens.Jwt.JwtPayload`
Moving this function to the enum would not be helpful as separate functions would need to be created there as well